app/controllers/live_controller.rb

```diff
@@ -0,0 +1,17 @@
+class LiveController < WebsocketRails::BaseController
+  def initialize_session
+    # perform application setup here
+    controller_store[:message_count] = 0
+  end
+
+  def contestant_clicked
+    new_message = {:message => "contestant clicked"}
+    WebsocketRails.users[1].send_message(:contestant_clicked, {:message => "you\'ve got an upvote "})
```

Comment:
Line is too long. [101/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.